scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+- group: system_monitor
+  name: heartbeat_period_milliseconds
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: Period for sending the SBP_HEARTBEAT and MSG_BASE_POS SBP messages
+  Notes:
+- group: telemetry_radio
+  name: configuration_string
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: Configuration string to send radio modem over UART when detected
+  Notes:
+- group: uart_ftdi
+  name: mode
+  type: enum
+  units:
+  default value: SBP
+  enumerated possible values: SBP,NMEA,RTCM
+  Description: Configure mode for uart for the USB port on Piksi
+  Notes:
+- group: uart_ftdi
+  name: sbp_message_mask
+  type: integer
+  units:
+  default value: 65535 (decimal), 0xFFFF (hex)
+  enumerated possible values:
+  Description: Configure the message mask for SBP messages on the uart for the USB
+    port on Piksi
+  Notes: 'The message mask is bitwise anded to the message identifier for a particular
+    message.  If the result is non-zero, the message will be sent over this UART.
+    For example, consider the Piksi firmware sending an SBP
+    message with ID 0x0041. If UART A has mask "64" (0x0040),
+    The SBP subsystem bitwise-ands the message id with the UART A mask giving the result
+    of 0x0040. Since the result is non-zero, the message is valid for UART A and is sent.
+    Practically, the UART with mask 64 (0x0020) transmits
+    only RTK observation data and the USART with mask 65280 (0xFF00) transmits
+    most messages (this is the default USB USART on Piksi).'
+- group: uart_ftdi
+  name: baudrate
+  type: integer
+  units: bits/s
+  default value: '115200'
+  enumerated possible values:
+  Description: The baudrate for the uart for the USB port on Piksi
+  Notes:
+- group: uart_uarta
+  name: mode
+  type: enum
+  units:
+  default value: SBP
+  enumerated possible values: SBP,NMEA,RTCM
+  Description: Configure mode for uart
+  Notes:
+- group: uart_uarta
+  name: sbp_message_mask
+  type: integer
+  units:
+  default value: 64 (decimal), 0x0020 (hex)
+  enumerated possible values:
+  Description: Configure the message mask for SBP messages on uart
+  Notes: The default message mask on this uart (0x0020) is appropriate for a radio to communicate observation messages
+    to another Piksi.
+- group: uart_uarta
+  name: configure_telemetry_radio_on_boot
+  type: boolean
+  units:
+  default value: 'TRUE'
+  enumerated possible values: true,false
+  Description: Determines whether this uart will attempt to configure a telemetry
+    radio upon boot
+  Notes: If a telemetry radio is connected to this UART, this should be set to true
+    in order to send the configuration string to the radio.
+- group: uart_uarta
+  name: baudrate
+  type: integer
+  units: bits/s
+  default value: '115200'
+  enumerated possible values:
+  Description: The baudrate for the uart
+  Notes: The radio baudrate may be constrained by the particular RF equipment used
```

Comment:
UART A may not always be used with the radio, that just the out of the box config.